### ISSUE TITLE
Create Layout for New Toolkit Page

### ISSUE THREAD

**Issue by jdingeman** [hackforla/website#3959]
### Dependency 
- [x] https://github.com/hackforla/website/issues/3744

### Overview
We will be implementing a new Toolkit page to replace the one that is currently on the published website. To do so, you will need to work off of the `feature-homepage-launch` branch. See [How to work off of a feature branch](https://github.com/hackforla/website/wiki/How-to-work-off-of-a-feature-branch)

### Action Items
- [ ] See Figma [design](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=24519-284897&t=XgEjXvcOIPMGrMnK-0) in the red box with an `issue 3626` label
  - [ ] You will be creating the layout as shown in screenshot below
- [ ] Create a new file in `website/pages` called `our-toolkit.html` and implement the design
- [ ] You can use the [current toolkit](https://github.com/hackforla/website/blob/gh-pages/pages/toolkit.html) as a reference for the new page. 
- [ ] For the dropdown filter, try to use the reusable component made in https://github.com/hackforla/website/issues/3744 when that issue is merged
  - If you are not able to implement it, please provide a write up in the comments with the blockers and what might be preventing it from being used
  - If you are able to implement it, please provide a write up in the comments how you implemented this
  - Your write up will be used later in the Wiki 

<details>
<summary>Proposed Figma design</summary>

![image](https://user-images.githubusercontent.com/88953806/231903632-bc18273a-963e-4c28-ac28-a35a8b5b2b7a.png)

</details>

### Resources/Instructions
- [How to work off of a feature branch](https://github.com/hackforla/website/wiki/How-to-work-off-of-a-feature-branch)
- Figma design [page](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=24519-284897&t=XgEjXvcOIPMGrMnK-0)
- Published [Toolkit page](https://www.hackforla.org/toolkit/)
- Using [Liquid in Jekyll](https://jekyllrb.com/docs/liquid/)

**Comment by plang-psm** [comment]
i. Availability: M-F Flexible
ii. ETA: 3/4/23

**Comment by plang-psm** [comment]
@jdingeman Is there also a mobile design to reference as well? I checked the Figma document provided but only found desktop views available.

**Comment by vanessavun** [comment]
Availability: weekday evenings
ETA: 3/10/2023

**Comment by jdingeman** [comment]
> @jdingeman Is there also a mobile design to reference as well? I checked the Figma document provided but only found desktop views available.

I think that may be up for interpretation and can reflect how the current toolkit's mobile view is displayed, but we should discuss that with product.

**Comment by vanessavun** [comment]
Something urgent came up and I won't be able to finish this next week.

**Comment by vanessavun** [comment]
Availability: weekday evenings
ETA: 3/23/2023

**Comment by vanessavun** [comment]
Progress: I created the HTML file with similar code from the current toolkit. Currently studying the current toolkit file and SCSS to see how to create the layout. What is left to do is creating the navigation table on the left side.
Blockers: None
Availability: Weekday nights, mostly Fridays.
ETA: Between 3/23 - 3/30.

**Comment by vanessavun** [comment]
I'm unable to finish this project within the ETA. I'm bogged down on an internship project.

**Comment by christinaor** [comment]
Availability: Weekdays 8PM-10PM EST
ETA: 3/28 - 3/31

**Comment by christinaor** [comment]
Progress: I started rearranging pieces of the HTML file and adjusting the SCSS files based on the Figma provided. I was previously blocked by a layout issue due to my own understanding of the current layout of toolkit.html, which has since been resolved. At office hours last night, I had my questions about a hidden button clarified and anticipate needing to redo the layout, so I may need a few more days to complete this.
Blockers: None now.
Availability: Weekdays 8PM-10PM EST
ETA: 3/30 - 4/3

**Comment by christinaor** [comment]
Progress: Made progress on fixing the layout of the cards with the search bar above last night.
Blocker: May pause on working on this - looking for confirmation that toolkit work is on hold, possibly due to a dependency for the dropdown ( #3985 ).

**Comment by jdingeman** [comment]
Hey @christinaor - moving this to the Ice Box for now while the dependency is being worked on, but because the issue is almost complete, I don't want you to lose the progress you made on this issue. I will unassign you while it's in the Ice Box but I will reach out again via Slack to let you know when the issue is in the Prioritized Backlog and assign you if you can take it up again.

**Comment by JessicaLucindaCheng** [comment]
- Edit the link for Figma design to be this in the red box: https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=24519-284897&t=XgEjXvcOIPMGrMnK-0

**Comment by jdingeman** [comment]
- This issue is already covered by
  - https://github.com/hackforla/website/issues/3985
  - https://github.com/hackforla/website/issues/4136
